randovania/data/json_data/prime1.txt

```diff
@@ -1160,105 +2161,192 @@ Twin Fires Tunnel
 Asset id: 3835971118
 > Dock to Twin Fires; Heals? False
   * Normal Door to Twin Fires/Dock to Twin Fires Tunnel
+  > Dock to Transport to Tallon Overworld West
+      Any of the following:
+          Morph Ball and Boost Ball and Heat-Resisting Suit
```

Comment:
This should be Spider Ball not Boost Ball

Reply:
Actually that whole and block was kinda unnecessary; I don't know *why* that's there.

Reply:
It makes sense to have. Otherwise you'd need Space Jump to cross which isn't the case